docs/UserGuide.md

```diff
@@ -235,6 +236,8 @@ e.g. `edit 1 t/CS2102 t/CS2102` will only register t/CS2102 once!
 
 :man_technologist: **Congratulations! You have completed a task. Now, you can remove it by using this command.**
 
+:man_technologist: **Think twice before using this. This action is not reversible.**
```

Comment:
Would removing all mentions of action not being reversible be clearer for the user? Even though the issue raised was for clear and delete, the same can be said about most other commands since we do not have an undo feature. Otherwise, would it be better to add this line at the start of section 3 when we are introducing the command format?

Reply:
I'm not really sure where the best place is to put the warning. But I have removed the warnings in delete and clear.